### PR TITLE
Adding Flag for removing stopwords

### LINKED ISSUE #33: Remove functional words using a list of stopwords
**Is your feature request related to a problem? Please describe.**
The English language has a lot of short connector words which have a grammatical function but don't necessarily provide meaning. Thus they can be omitted when creating abbreviations.

**Describe the solution you'd like**
A simple solution would be to save a list of stopwords and to have a flag that the user could use to remove them.

Here is an example:
`$ abbreviate snake "The strategy of limited science" --stopwords
tstg_ltd_sci`

versus the current behavior (so without the flag):
`$ abbreviate snake "The strategy of limited science" --stopwords
the_stg_of_ltd_sci`

**Additional context**
Initially we could curate our own list of stopwords. Eventually we could use this as a possible source:
https://github.com/bbalet/stopwords

### PR DESCRIPTION
**Pull request by leafoliage**
# Description

This PR adds a flag `--stopwords`, which removes stopwords from abbreviation.

Fixes #33

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

I’ve created a test for the `removeStopwords` function.

- [x] Running existing tests
- [x] Created new tests

# Checklist:

- [x] My code has been linted (`make lint`)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have rebased my branch to include the latest changes from `master`

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -126,6 +127,7 @@ func init() {
 	rootCmd.PersistentFlags().IntVarP(&optMax, "max", "m", optMax, "Maximum length of string, keep on abbreviating while the string is longer than this limit")
 	rootCmd.PersistentFlags().BoolVarP(&optFrmFront, "from-front", "", optFrmFront, "Shorten from the front")
 	rootCmd.PersistentFlags().StringVarP(&optStrategy, "strategy", "", optStrategy, "Abbreviation strategy")
+	rootCmd.PersistentFlags().BoolVarP(&optRemoveStopwords, "stopword", "", optRemoveStopwords, "Remove stopwords from abbreviation")
```
Comment (dnnrly) [inline cmd/root.go:130]: Could this option be called `no-stopwords`? I've tried to keep with GNU style options in this codebase.

**Review by dnnrly — COMMENTED** [review]
I like this PR! Just the one change as far as I can see. I'll try and kick the CI teats in to life too.

File: cmd/root.go (hunk 1 above)
Comment (leafoliage) [reply cmd/root.go]: Okay! I've renamed it to `no-stopwords`

**Comment by dnnrly** [comment]
Thanks for this new feature @leafoliage - and apolgies for taking so long to merge it. As is tradition, if you do twitter then please let me know your handle and I will git you a follow. :smile:

**Comment by leafoliage** [comment]
@dnnrly Thank you too for approving this PR! Here's my twitter handle: `leafoliage`

**Pull request merged**